src/main/java/net/sf/jabref/gui/groups/GroupDialog.java

```diff
@@ -77,37 +69,37 @@
-    private final JRadioButton m_explicitRadioButton = new JRadioButton(
+    private final JTextField nameField = new JTextField(GroupDialog.TEXTFIELD_LENGTH);
+    private final JRadioButton explicitRadioButton = new JRadioButton(
             Localization.lang("Statically group entries by manual assignment"));
-    private final JRadioButton m_keywordsRadioButton = new JRadioButton(
+    private final JRadioButton keywordsRadioButton = new JRadioButton(
             Localization.lang("Dynamically group entries by searching a field for a keyword"));
-    private final JRadioButton m_searchRadioButton = new JRadioButton(
+    private final JRadioButton searchRadioButton = new JRadioButton(
             Localization.lang("Dynamically group entries by a free-form search expression"));
-    private final JRadioButton m_independentButton = new JRadioButton(
+    private final JRadioButton independentButton = new JRadioButton(
             Localization.lang("Independent group: When selected, view only this group's entries"));
-    private final JRadioButton m_intersectionButton = new JRadioButton(
+    private final JRadioButton intersectionButton = new JRadioButton(
             Localization.lang("Refine supergroup: When selected, view entries contained in both this group and its supergroup"));
-    private final JRadioButton m_unionButton = new JRadioButton(
+    private final JRadioButton unionButton = new JRadioButton(
             Localization.lang("Include subgroups: When selected, view entries contained in this group or its subgroups"));
     // for KeywordGroup
-    private final JTextField m_kgSearchField = new JTextField(GroupDialog.TEXTFIELD_LENGTH);
-    private final TextField m_kgSearchTerm = new TextField("keywords", "", false);
-    private final JCheckBox m_kgCaseSensitive = new JCheckBox(Localization.lang("Case sensitive"));
-    private final JCheckBox m_kgRegExp = new JCheckBox(Localization.lang("regular expression"));
+    private final JTextField kgSearchField = new JTextField(GroupDialog.TEXTFIELD_LENGTH);
+    private final TextField kgSearchTerm = new TextField("keywords", "", false);
+    private final JCheckBox kgCaseSensitive = new JCheckBox(Localization.lang("Case sensitive"));
+    private final JCheckBox kgRegExp = new JCheckBox(Localization.lang("regular expression"));
     // for SearchGroup
-    private final JTextField m_sgSearchExpression = new JTextField(GroupDialog.TEXTFIELD_LENGTH);
-    private final JCheckBox m_sgCaseSensitive = new JCheckBox(Localization.lang("Case sensitive"));
-    private final JCheckBox m_sgRegExp = new JCheckBox(Localization.lang("regular expression"));
+    private final JTextField sgSearchExpression = new JTextField(GroupDialog.TEXTFIELD_LENGTH);
+    private final JCheckBox sgCaseSensitive = new JCheckBox(Localization.lang("Case sensitive"));
+    private final JCheckBox sgRegExp = new JCheckBox(Localization.lang("regular expression"));
```

Comment:
Not convinced. As `sg` is not really that common. Why not searchGroupRegExCheckBox?